docs/pkgdown.yml

```diff
@@ -0,0 +1,25 @@
+pandoc: 2.11.4
+pkgdown: 2.0.3
+pkgdown_sha: ~
+articles:
```

Comment:
It's not clear to me from reading the documentation that this file should live in the `doc` folder? Was that specified somehwhere or is it created automtically in this folder when building the site?

Reply:
Yes, the documentation is not clear. It is generated when building the site. It took me some time to find it.

I tested it in admiraltemplate and it works if it is the `docs` folder.